src/early_multicellular_stage/editor/ReproductionOrder.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Godot;
+
+/// <summary>
+///   Handles showing and changing the order in which cells in an early multicellular creature will divide.
+/// </summary>
+public class ReproductionOrder : MarginContainer
```

Comment:
```suggestion
public class ReproductionOrderEditor : MarginContainer
```

I think this kind of name would be a bit better to not confuse this with a pure data class that would just hold the order things divide in.

Reply:
Reading the changes in CellBodyPlanEditorComponent I now see that, this is just a *single* line in a reproduction order. So this should have a much different name. I think also that it would be worth it to split a new class ReproductionOrderEditor from the body plan editor to simplify that class (preferring composition of GUI elements rather than always adding more and more functionality to a single GUI class). Then this class would be named something like ReproductionOrderCellEntry or something like that to signify that this is just a single element.